### DISCUSSION TITLE
[draft-v1] Section 2.2.2: Persistent Storage

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#54] [Text comments] [answered]
> Table may also read from and write to ASDF

I think we want to clarify that we are going to deprecate `astropy.io.misc.asdf` in `astropy` v5.1 because it has been moved to a new package called `asdf-astropy`,

cc @WilliamJamieson

**Comment by WilliamJamieson** [comment, marked as answer]
> I think we want to clarify that we are going to deprecate `astropy.io.misc.asdf` in `astropy` v5.1 because it has been moved to a new package called `asdf-astropy`

Yes this is probably a good idea. I footnote to the effect of
```latex
\footnote{Starting in v5.1, direct astropy support for ASDF has been depreciated in favor of moving this
functionality to the new asdf-astropy package. This has been done to accommodate the differing release
schedules between ASDF and astropy.}
```

**Comment by pllim** [comment]
@WilliamJamieson , should I cite `asdf-astropy` in any way?

**Reply by WilliamJamieson** [reply]
@perrygreenfield and @nden should this be done? If so, how?

**Comment by pllim** [comment]
OK, let's move discussion to #60 then?
